app/src/interfaces/input-block-editor/editorjs-overrides.css

```diff
@@ -612,4 +621,4 @@
 
 .codex-editor .ce-inline-tool--active {
 	color: var(--primary);
-}
+}
```

Comment:
```suggestion
}

```
Should contain an empty line according to our prettier formatting